crypto/src/random/mod.rs

```diff
@@ -180,7 +180,7 @@ impl<B: StarkField, H: Hasher> RandomCoin<B, H> {
     where
         E: FieldElement<BaseField = B>,
     {
-        for _ in 0..200 {
+        for _ in 0..500 {
```

Comment:
Let's bump it up to 1000, and also let's update the documentation comment above.

Reply:
Updated!